module/EntraBeta/AdditionalFunctions/Update-EntraBetaUserAuthenticationRequirement.ps1

```diff
@@ -0,0 +1,52 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.
+#  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+function Update-EntraBetaUserAuthenticationRequirement {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, HelpMessage = "Enter the User ID (ObjectId or UserPrincipalName) of the user whose authentication requirements you want to update.")]
+        [Alias("ObjectId")]
+        [System.String] $UserId,
+
+        [Parameter(Mandatory = $true, HelpMessage = "Specify the per-user MFA state. Valid values are 'enabled', 'disabled', 'enforced' or 'unknownFutureValue'.")]
```

Comment:
`unknownFutureValue` is a value added to `Evolvable Enums` in Graph. I don't think it should supported here.
https://github.com/microsoft/api-guidelines/blob/vNext/graph/patterns/evolvable-enums.md

Reply:
Addressed.